### ISSUE TITLE
Insphpect results

### ISSUE THREAD

**Issue by HenkPoley** [eislambey/php-rsmq#4]
https://insphpect.com/report/5f168d883467d

I just randomly chose a PHP project from my main Github page. Happened to be this one 🤷‍♂️

The tool seems to mope about usage of `new Unit()` in Islambey\RSMQ\RSMQ. After careful inspection, all the 'utils' could be static functions. There is not state (no `$this`) in Islambey\RSMQ\Util. Maybe use them like that?

Edit: What do you know. It will also tell you not to use static functions. [Example](https://insphpect.com/report/class/5f169284b446b/WoohooLabs%5CYin%5CJsonApi%5CSchema%5CError%5CError)

**Comment by eislambey** [comment]
Yes, the `Util` class functions can be static. But you are saying this is bad to.
Maybe we can have pure functions or we can inject Util class but this will be BC break. What is your opinion?
